Serializer/BaseSerializerHandler.php

```diff
@@ -27,6 +27,11 @@
     protected $manager;
 
     /**
+     * @var string[]
+     */
+    protected static $formats = array('json', 'xml', 'yml');
```

Comment:
> Both, I think.

Oh, I copied pasted but forgot to change the code. After the 'Or Just' the code should be like this: 
```php
BaseSerializerHandler::setFormats(array('json', 'xml', 'yml'));
```
But anyway I got my answer.
> not sure this is possible though

As I know this is not possible. But I will add the deprecating message anyway.